backend/src/test/java/com/redhat/cloud/notifications/routers/ValidationEndpointTest.java

```diff
@@ -0,0 +1,81 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.MockServerClientConfig;
+import com.redhat.cloud.notifications.MockServerConfig;
+import com.redhat.cloud.notifications.TestHelpers;
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.models.EventType;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectMock;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.smallrye.mutiny.Uni;
+import org.junit.jupiter.api.Test;
+
+import static com.redhat.cloud.notifications.MockServerClientConfig.RbacAccess.FULL_ACCESS;
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+@QuarkusTest
+class ValidationEndpointTest {
+
+    @MockServerConfig
+    MockServerClientConfig mockServerConfig;
+
+    @InjectMock
+    ApplicationResources appResources;
+
+    @Test
+    void shouldReturnNotFoundWhenTripleIsInvalid() {
```

Comment:
The mocked `ApplicationResources` should throw a `NoResultException` when the event type is not found to match the runtime code.